src/js/components/organisms/tabbed-content/tabbed-content.models.ts

```diff
@@ -0,0 +1,3 @@
+/** @format */
+
+export interface TabbedContentProps {}
```

Comment:
Remove this file or import the model definition here.

Reply:
True, need to do that. Forgot about it.